command/agent/testagent.go

```diff
@@ -0,0 +1,278 @@
+package agent
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/consul/consul"
+	"github.com/hashicorp/consul/consul/structs"
+	"github.com/hashicorp/consul/logger"
+	"github.com/hashicorp/consul/testutil/retry"
+	"github.com/hashicorp/consul/types"
+	"github.com/hashicorp/consul/version"
+	uuid "github.com/hashicorp/go-uuid"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano()) // seed random number generator
```

Comment:
We've got https://github.com/hashicorp/consul/tree/master/vendor/github.com/sean-/seed vendored and can use that, though this is probably cheaper and fine for unit tests.